Tests/Tests/CPSSpec.swift

```diff
@@ -0,0 +1,128 @@
+import SwiftCheck
+import XCTest
+import Runes
+
+
+class CPSSpec: XCTestCase {
+    func testFunctor() {
+        // fmap id = id
+        property("identity law") <- forAll { (x: CPSOf<Int>) in
+            let lhs = id <^> x.getAsync
+            let rhs = x.getAsync
+
+            return lhs == rhs
+        }
+
+        // fmap (f . g) = (fmap f) . (fmap g)
+        property("function composition law") <- forAll { (o: CPSOf<Int>, fa: ArrowOf<Int, Int>, fb: ArrowOf<Int, Int>) in
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 121 characters (line_length)